soc/xtensa/intel_adsp/tools/cavstool.py

```diff
@@ -343,7 +537,10 @@ def ipc_command(data, ext_data):
         dsp.HIPCIDR = (1<<31) | ext_data
 
 async def main():
-    global hda, sd, dsp, hda_ostream_id
+    log.debug("Removing old hugetables")
+    runx("rm -rf /dev/hugepages/cavs_*")
+    #TODO this bit me, remove the globals, write a little FirmwareLoader class or something to contain.
```

Comment:
Agreed.

Reply:
Todo in a cleanup round, when I move things to use HDAStream perhaps